### PR TITLE
gpg: remove temp dir, change env_add_path to bin, and fix license identifier

### PR DESCRIPTION
**Pull request by dodorz**
remove $PLUGINSDIR dir

### CODE REVIEW

File: bucket/gpg.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "license": "GPL-3.0",
     "url": "https://www.gnupg.org/ftp/gcrypt/binary/gnupg-w32-2.2.15_20190326.exe#/dl.7z",
     "hash": "sha1:f5f7eeadfdf6ad971b5313f045083b4f95ace07b",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse",
     "env_add_path": "bin",
```
Comment (Ash258) [inline bucket/gpg.json]: Change `env_add_path` to `bin` with list of all exe files.

```powershell
    Directory: C:\Users\Cabera.Jakub\Downloads\gnupg-w32-2.2.15_20190326\bin

Mode                LastWriteTime         Length Name
----                -------------         ------ ----
-a----        26/03/2019    14:53         627712 dirmngr.exe
-a----        26/03/2019    14:53          88064 dirmngr_ldap.exe
-a----        26/03/2019    14:53        1125888 gpg.exe
-a----        26/03/2019    14:53         412160 gpg-agent.exe
-a----        26/03/2019    14:53         151040 gpgconf.exe
-a----        26/03/2019    14:53         170496 gpg-connect-agent.exe
-a----        26/03/2019    14:54          16896 gpgme-w32spawn.exe
-a----        26/03/2019    14:53          81408 gpg-preset-passphrase.exe
-a----        26/03/2019    14:53         509440 gpgsm.exe
-a----        26/03/2019    14:53         162816 gpgtar.exe
-a----        26/03/2019    14:53         521728 gpgv.exe
-a----        26/03/2019    14:53         219648 gpg-wks-client.exe
-a----        26/03/2019    14:54          67072 pinentry-basic.exe
-a----        26/03/2019    14:53         402944 scdaemon.exe
```

File: bucket/gpg.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "license": "GPL-3.0",
```
Comment (Ash258) [inline bucket/gpg.json]: Wrong identifier. use `GPL-3.0-only` and add description property

**Comment by h404bi** [comment]
Please also add `description` field for this manifest. :D

**Comment by rasa** [comment]
Has anyone tested if moving to a shim, will break gpg-agent’s functionality?

**Pull request closed without merging**